MailSenderTask2/src/main/java/com/demo/service/MailSenderService.java

```diff
@@ -0,0 +1,17 @@
+package com.demo.service;
+
+import java.util.List;
+import com.demo.model.MailSender;
+
+public interface MailSenderService {
```

Comment:
In ideal case - you would have to create single project with single interface and with 2 implementations.
Idea is to switch between implementations on the spot.